packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1805,42 +1782,78 @@ describe("calcite-combobox", () => {
       combobox.setProperty("selectionMode", selectionMode);
       combobox.setProperty("allowCustomValues", allowCustomValues);
       const inputEl = await page.find(`#myCombobox >>> input`);
-      const buttonEl = await page.find("calcite-button");
 
       await inputEl.focus();
       await page.waitForChanges();
       expect(await page.evaluate(() => document.activeElement.id)).toBe("myCombobox");
+
+      const comboboxRect = await page.evaluate(() => {
+        const comboboxEl = document.querySelector("#myCombobox");
+        return comboboxEl.getBoundingClientRect().toJSON();
+      });
+
       await inputEl.type("asdf");
       await page.waitForChanges();
-
-      if (allowCustomValues) {
-        await inputEl.press("Enter");
-        await buttonEl.click();
-        await page.waitForChanges();
-        expect(await page.evaluate(() => document.activeElement.id)).toBe("button");
-        expect(await combobox.getProperty("value")).toBe("asdf");
-      } else {
-        await buttonEl.click();
-        await page.waitForChanges();
-        expect(await page.evaluate(() => document.activeElement.id)).toBe("button");
-        expect(await combobox.getProperty("value")).toBe("");
-      }
+      await page.mouse.move(10, 2 * comboboxRect.bottom);
+      await page.mouse.down();
+      await page.waitForChanges();
+      await page.mouse.up();
+      await page.waitForChanges();
+      expect(await page.evaluate(() => document.activeElement.id)).not.toBe("myCombobox");
+      expect(await inputEl.getProperty("value")).toBe("");
+      expect(await combobox.getProperty("value")).toBe(allowCustomValues ? "asdf" : "");
```

Comment:
Nitpick: can you use a word vs `asdf`? I think the spelling linter will warn otherwise.

Applies to the test below.